MonoGame.Framework/Graphics/SpriteBatch.cs

```diff
@@ -125,11 +125,9 @@ void Setup()
 			var vp = gd.Viewport;
 
 		    Matrix projection;
-#if PSM || DIRECTX
             Matrix.CreateOrthographicOffCenter(0, vp.Width, vp.Height, 0, -1, 0, out projection);
```

Comment:
Yes - but only since I made that change to fixup the depth values in GLSL, such that depths output from GLSL match the Direct3D convention. As soon as I made that change, it made this fixup in SpriteBatch both redundant and incorrect.